.gitignore

```diff
@@ -2,3 +2,6 @@ target/
 /test-data
 /.bsp
 /image/*.zip
+project/
```

Comment:
The `/project` dir contains at least two files that have to be checked-in: `build.properties` and `plugins.sbt`.

SBT is creating a `/project/target` and `/project/project/target` directories for me (yuck!) but they are both ignored thanks to the `target/` rule at the top.

Reply:
I've corrected that to global .gitignore, without including /project